lib/shopify_app/scripttags_manager.rb

```diff
@@ -25,6 +25,8 @@ def create_scripttags
       return unless required_scripttags.present?
 
       required_scripttags.each do |scripttag|
+        # src may be call()able and we don't want to overwrite it
+        scripttag = scripttag.merge(src: build_src(scripttag[:src]))
```

Comment:
This will modify the callers Hash and, depending on how the manager is called, may overwrite the callable object in the global scripttag config.